docs/en/connector-v2/sink/Paimon.md

```diff
@@ -43,7 +43,7 @@ libfb303-xxx.jar
 | data_save_mode              | Enum   | No       | APPEND_DATA                  | The data save mode                                                                                                                                               |
 | paimon.table.primary-keys   | String | No       | -                            | Default comma-separated list of columns (primary key) that identify a row in tables.(Notice: The partition field needs to be included in the primary key fields) |
 | paimon.table.partition-keys | String | No       | -                            | Default comma-separated list of partition fields to use when creating tables.                                                                                    |
-| paimon.table.write-props    | Map    | No       | -                            | Properties passed through to paimon table initialization, [reference](https://paimon.apache.org/docs/0.8/maintenance/configurations/#coreoptions).               |
```

Comment:
This link seems to be accessible

Reply:
> This link seems to be accessible

I found that this link will only keep the latest version. For example, when version 0.9 is released, it will overwrite 0.8.

Reply:
I agree with your viewpoint